crates/cli-support/src/js/mod.rs

```diff
@@ -614,16 +614,19 @@ impl<'a> Context<'a> {
         } else {
             declare_or_export = "export";
 
-            sync_init_function.push_str(&format!("\
+            sync_init_function.push_str(&format!(
+                "\
+                {declare_or_export} type SyncInitInput = BufferSource | WebAssembly.Module;\n\
                 /**\n\
-                * Synchronously compiles the given `bytes` and instantiates the WebAssembly module.\n\
+                * Uses a given `module` directly or compiles it from a BufferSource and then\n\
+                * instantiates the WebAssembly module.\n\
```

Comment:
This is a bit unclear; it makes it sound like the module doesn't get instantiated if it's precompiled. 

I think ```Instantiates the given `module`, which can either be bytes or a precompiled `WebAssembly.Module`.``` is a bit clearer.